src/main/scala/it/pagopa/interop/agreementprocess/api/impl/AgreementApiServiceImpl.scala

```diff
@@ -35,13 +49,25 @@ final case class AgreementApiServiceImpl(
   tenantManagementService: TenantManagementService,
   attributeManagementService: AttributeManagementService,
   authorizationManagementService: AuthorizationManagementService,
-  fileManager: FileManager
+  partyManagementService: PartyManagementService,
+  userRegistry: UserRegistryService,
+  pdfCreator: PDFCreator,
+  fileManager: FileManager,
+  offsetDateTimeSupplier: OffsetDateTimeSupplier,
+  uuidSupplier: UUIDSupplier
 )(implicit ec: ExecutionContext)
     extends AgreementApiService {
 
   implicit val logger: LoggerTakingImplicit[ContextFieldsToLog] =
     Logger.takingImplicit[ContextFieldsToLog](this.getClass)
 
+  private[this] val agreementTemplate = Source
+    .fromResource("agreementTemplate/index.html")
+    .getLines()
+    .mkString(System.lineSeparator())
+
+  private val agreementDocumentSuffix: String = "_richiesta_di_fruizione.pdf"
```

Comment:
```suggestion
  private val agreementDocumentSuffix: String = "_agreement_contract.pdf"
```
it would be easier for newcomers to understand which kind of object this is